common/static/common/templates/discussion/inline-discussion.underscore

```diff
@@ -23,3 +23,8 @@
     </div>
 </section>
 
+<script>
+$('.btn-link.new-post-btn').on("click", function() {
+     $('.forum-new-post-form .thread-title').focus()
+ });
+</script>
```

Comment:
Please do these changes on `mcka-theme` as we should not do the change on Edx core files.